test/unit/spec/util/sdp/index.js

```diff
@@ -490,6 +491,222 @@ a=ssrc:0000000000 label:d8b9a935-da54-4d21-a8de-522c87258244\r
   });
 });
 
+describe('unifiedPlanFilterLocalCodecs', () => {
+  it('should filter codecs in a local SDP based on those advertised in the remote SDP', () => {
```

Comment:
You could add few more unittests for the -ve cases where we do not expect any updates to the offer.sdp

Reply:
I am sure you know this - but running `npm run cover` would give you an idea of missing code paths.